### ISSUE TITLE
Remove microphone indicator

### ISSUE THREAD

**Issue by DmitriyMaximov** [Kagami/vmsg#19]
Microphone indicator is not hidden after recording. 
It is described here https://stackoverflow.com/questions/42786056/removing-microphone-indicator-in-google-chrome

In vmsg we need save stream in initAudio() method:
```
getUserMedia({audio: true}).then(stream => {
	  this.stream = stream;
...
```
 and stop tracks in close() method, I think.
Like this (close):
`if (this.stream) this.stream.getTracks().forEach(track => track.stop());`

**Comment by Kagami** [comment]
This should be fixed in #17.

**Comment by DmitriyMaximov** [comment]
I'm sorry, I was looking at the npm package. Can you update the npm package?

**Comment by Kagami** [comment]
Sure, published 0.3.5.

**Comment by DmitriyMaximov** [comment]
Super! Thank you very much!
